index.html

```diff
@@ -296,10 +296,8 @@ <h2>
           The steps to <dfn>launch a web app with handling</dfn> are given by
           the following algorithm. The algorithm takes a
           <a data-cite="appmanifest#dfn-processed-manifest">processed manifest</a>
-          |manifest:processed manifest|,
-          an optional |target URL:URL| or {{LaunchParams}}
-          |params:LaunchParams|, and returns a
-          [=service worker client/window client=].
+          |manifest:processed manifest| and an optional |target URL:URL| or
```

Comment:
This almost sounds as if the algorithm will only have one of a target url or LaunchParams

Reply:
That is the intention though it might be a bit dodgy to do.

Reply:
Reworded.